### PR TITLE
merging with Alkalus 5.08

### PR DESCRIPTION
**Pull request by e99999**
merging with Alkalus 5.08

### CODE REVIEW

**Comment by diregarath** [comment]
@e99999 Looks like you were trying to remove the idea files and logs :)  👍

**Comment by leagris** [comment]
Please remove https://github.com/bear989/GT5u-Bears-Den/pull/3/commits/138985c8667ca7720d27ebcac81cd0bbf47cce1a as this belong to a texture/resource pack as end-user preference.

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (diregarath) [inline .classpath]: Does this need to be committed?  Seems like an IDE file.

File: .gitignore [hunk 2]
```diff
@@ -1,73 +1,10 @@
...
+*.log
 .idea_modules/
-
-/out/
-eclipse/
-*.bat
+*.xml
+*.log
+*.iml
+*.log
```
Comment (diregarath) [inline .gitignore]: Do we need *.log twice?

File: .project [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (diregarath) [inline .project]: IDE file?

File: build.gradle [hunk 4]
```diff
@@ -28,9 +28,6 @@ version = "${config.gt.version}"
 group= "gregtech"
 archivesBaseName = "gregtech"
 
-sourceCompatibility = 1.8
```
Comment (diregarath) [inline build.gradle]: Shouldn't we build against java 8?  why not?

File: build.properties [hunk 5]
```diff
@@ -1,11 +1,11 @@
 minecraft.version=1.7.10
-forge.version=10.13.4.1614-1.7.10
```
Comment (diregarath) [inline build.properties]: I switched this to 1614 - I would like to keep it on the last version of 1.7.10 forge.

File: compile_Gregtech.cmd [hunk 6]
```diff
@@ -0,0 +1,3 @@
+@echo on
```
Comment (diregarath) [inline compile_Gregtech.cmd]: What is this for?  gradlew build is easy enough to run.

File: gradlew [hunk 7]
```diff
@@ -1,164 +0,0 @@
-#!/usr/bin/env bash
```
Comment (diregarath) [inline gradlew]: Why are we removing gradlew and adding the bat version? I often develop on os x and linux (and cygwin on windows) - this needs to be restored.  We should have both.

File: .gitignore (hunk 2 above)
Comment (diregarath) [reply .gitignore]: Any reason we are removing the comments that make it easier to understand the gitignore?

File: .gitignore [hunk 8]
```diff
@@ -1,73 +1,6 @@
-### Windows ###
```
Comment (diregarath) [inline .gitignore]: I liked how this was organized so we knew why specific things were being excluded.  We should bring it back if possible.

**Review by diregarath — CHANGES_REQUESTED** [review]
Please don't merge this until at least gradlew is back, we are building java 1.8 (or do we have a reason to stay with java 7?), and we are using forge 10.13.4.1614-1.7.10.

**Review by diregarath — APPROVED** [review]

**Pull request merged**